README.md

```diff
@@ -559,6 +560,250 @@ The model can also be downloaded and served locally.
 
    The output from the command will consist of a series of outputs from the model before and after training.
 
+### 🧪 Evaluate the newly trained model
+
+You can use the `ilab` command to evaluate the models you are training with several benchmarks. Currently, four benchmarks are supported.
+
+| Benchmark | Measures | Full Name | Description |
+| --- | --- | --- | --- |
+| MMLU | Knowledge | Massive Multitask Language Understanding | TODO |
+| MMLUBranch | Knowledge | N/A | TODO |
+| MTBench | Skills | Multi-turn Benchmark | TODO |
+| MTBenchBranch | Skills | N/A | TODO |
+
+> [!NOTE]
+> MTBench and MTBenchBranch use [prometheus-8x7b-v2.0](https://huggingface.co/prometheus-eval/prometheus-8x7b-v2.0) as the judge model by default. While you do not need to use this model as your judge, it is strongly recommended to do so if you have the nessessary hardware resources. You can download it via `ilab model download`.
```

Comment:
1) rewrap please
2) `necessary`

Reply:
Done - wrapping should probably be something enforcement by the markdown linter though, so it's consistent